packages/subiquity_client/lib/src/types.dart

```diff
@@ -4,6 +4,8 @@ import 'package:json_annotation/json_annotation.dart';
 part 'types.freezed.dart';
 part 'types.g.dart';
 
+// ignore_for_file: invalid_annotation_target
```

Comment:
```
info • The annotation 'JsonKey' can only be used on fields or getters • lib/src/types.dart:58:6 • invalid_annotation_target
...
```

https://pub.dev/documentation/freezed/latest/#disabling-invalid_annotation_target-warning-and-warning-in-generates-files